fedimint-server/src/atomic_broadcast/data_provider.rs

```diff
@@ -59,8 +59,8 @@ impl aleph_bft::DataProvider<UnitData> for DataProvider {
             return Some(UnitData::Signature(signature));
         }
 
-        // an empty vector is encoded in at most 8 bytes
-        let mut n_bytes = 8;
+        // the length of a vector is encoded in at most 9 bytes
+        let mut n_bytes = 9;
```

Comment:
This probably didn't cause the problem, right, but rather the other places in which the size wasn't added up correctly?

Reply:
I assume so